packages/@glimmer/application/src/builders/dom-builder.ts

```diff
@@ -0,0 +1,22 @@
+import { Cursor, NewElementBuilder, Environment } from "@glimmer/runtime";
+import { Builder } from "../application";
+
+export interface DOMBuilderOptions {
```

Comment:
This is a `Cursor` and the type is exposed from `@glimmer/runtime` as such. If you want a type alias for the same thing just `extend` it.

Reply:
@chadhietala Deny, `Cursor` does not allow `nextSibling` to be `undefined`. I don't think we want to make users have to explicitly set `nextSibling` to `null` every time.

Reply:
It is an `Option` in the constructor just do `{ element, nextSibling = null }`